ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomSheetDemoController.swift

```diff
@@ -81,14 +86,37 @@ class BottomSheetDemoController: UIViewController {
         bottomSheetViewController?.preferredExpandedContentHeight = 400
     }
 
-    private let personaListView: PersonaListView = {
+    private lazy var personaListView: UIScrollView = {
         let personaListView = PersonaListView()
         personaListView.personaList = samplePersonas
         personaListView.backgroundColor = Colors.NavigationBar.background
         personaListView.translatesAutoresizingMaskIntoConstraints = false
         return personaListView
     }()
 
+    private lazy var expandedContentView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(personaListView)
+
+        let bottomView = UIView()
+        bottomView.backgroundColor = .systemTeal
```

Comment:
not sure I understand the reason for setting the backgroundColor to .systemTeal here...

Reply:
Yeah sorry, the demo app design should be a bit more intentional. I'll change it to something better.

Reply:
I updated it, also added new screen recordings in the description.